RedMenu/RedMenuClient/menus/PlayerMenu.cs

```diff
@@ -26,6 +27,23 @@ public static void SetupMenu()
             menu.AddMenuItem(healBtn);
             menu.AddMenuItem(restoreStaminaBtn);
             menu.AddMenuItem(restoreEyeBtn);
+            menu.OnItemSelect += (_menu, _item, _index) =>
+            {
+                int index = (int)_index;
+                switch(index)
```

Comment:
Please do not switch based on index. Items will be added/removed based on permissions which changes the potential index value for each item. Use the actual item itself to check which item was selected.

Reply:
That makes sense, I did not consider this, will change.

Reply:
made new commit with if/else on the item instead of switch on index, I updated the descriptions to reflect only refilling cores. Been trying to play with the natives for the outer ring but no luck yet.

Reply:
It's something in here: https://vespura.com/doc/natives/#n_ATTRIBUTE for the outer cores. setting attributes index 0-2 and getting max attributes. I'll look into it more when i add the other options for it. I'm not going to make them items you can click on in the menu, it'll be a server config option that will allow setting max of everything if the server owner wants this enabled.